src/components/Profile/MyGymBookings.jsx

```diff
@@ -84,11 +84,11 @@ export default function MyGymBookings ({ loggedIn }) {
 
   /**
      * method that handles the edited booking
-     * @param {event} e
+     * @param @param {Integer} bookingId hold the booking id for the old booking
```

Comment:
This should probably specify that it is optional and that it will rebook the id not only hold an id.